conseil-api/src/main/resources/logback.xml

```diff
@@ -17,7 +17,7 @@
     </appender>
 
     <logger name="com.base22" level="TRACE"/>
-    <logger name="tech.cryptonomic.conseil.tezos.BigMapsOperations" level="${LORRE_BIG_MAPS_LOG_LEVEL:-OFF}"/>
+    <logger name="slick.jdbc.JdbcBackend.statement"  level="INFO"/>
```

Comment:
Not really. Previously there were 2 places to configure logging. For Slick it was `application.conf` (or `reference.conf`) for rest of the classes it was `logback.xml`. In my opinion logbook is a better place to configure logging (since you are configuring everything from single file, single place) I changed the logLevel in `reference.conf` and limited it at this stage. This is the reason for this line. Would you like me to revert it back as it was? I do not observe any difference in behavior at the end of the day, so I suppose it works as previously (for the end user).

Reply:
Now I understand the slick entry on the configuration file better... it attracted my attention but I didn't recall it well. Now it makes sense.

Fine for me.